src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -2707,54 +2707,69 @@ public async Task PostAsyncNonDuplex_ServerCompletesResponseBodyThenResetsStream
         [InlineData(true, HttpStatusCode.OK)]
         [InlineData(false, HttpStatusCode.OK)]
         [OuterLoop("Uses Task.Delay")]
-        public async Task SendAsync_ConcurentSendReceive_Ok(bool shouldWait, HttpStatusCode responseCode)
+        public async Task SendAsync_ConcurentSendReceive_Ok(bool shouldWaitForRequestBody, HttpStatusCode responseCode)
         {
-            TaskCompletionSource<bool> tsc = new TaskCompletionSource<bool>();
             string requestContent = new string('*', 300);
             const string responseContent = "SendAsync_ConcurentSendReceive_Ok";
-            var stream = new CustomContent.SlowTestStream(Encoding.UTF8.GetBytes(requestContent), tsc, trigger : 1, count : 10);
 
-            await Http2LoopbackServer.CreateClientAndServerAsync(async url =>
+            using (var server = Http2LoopbackServer.CreateServer())
```

Comment:
nit: shouldn't use 'var' in this case. it is not obvious what the type is.

Reply:
I'll do separate pass to cleanup other instances as well.